changelogs/current.yaml

```diff
@@ -116,6 +116,10 @@ minor_behavior_changes:
 - area: logging
   change: |
     changed category name for access log filter extensions to ``envoy.access_loggers.extension_filters``.
+- area: filter state
+  change: |
```

Comment:
Because the original change never made it into a release, please make the changelog reflect the net-change for this envoy release.

Reply:
The original PR did make it into 1.22 AFAICT. It's from 2/23.